### DISCUSSION TITLE
Whats the best way to configure volumes/mounts for synonyms?

### DISCUSSION THREAD

**Discussion by mattkallo** [Opster/opensearch-k8s-operator#525] [Q&A] [answered]
Is there a way to mount a volume to a specific path in the container? I am trying to configure a folder for the "synonyms_path". I saw couple of open issues related to allowing PVC on additional volumes (#484 ), but wanted to check if synonyms_path can be configured in some way.

**Comment by swoehrl-mw** [comment, marked as answer]
Hi @mattkallo. At the moment only secrets and configmaps can be added as additional volumes to the nodepools. Supporting PVCs and other types of volumes is still an open feature request.
So as a workaround, if you can store your synonyms file as a secret or configmap, you can mount it.
